packages/react/src/components/link/Link.tsx

```diff
@@ -6,11 +6,17 @@ import { IconLinkExternal } from '../../icons';
 import classNames from '../../utils/classNames';
 import { getTextFromReactChildren } from '../../utils/getTextFromReactChildren';
 
+export enum LinkSize {
+  Small = 'small',
+  Medium = 'medium',
+  Large = 'large',
+}
```

Comment:
Not actually related to this PR, but we could make one generic enum for sizes and use Pick<>. Instead of defining these again and again and think "should it be small or Small or S" 😄

Reply:
I guess we discussed this a bit earlier and decided to still go with separate still, in the future we might combine these.